packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.css

```
@@ -12,3 +12,17 @@ ion-card {
 geov-paginator {
   margin-left: auto;
 }
+
+.litteral-container {
+  overflow: clip;
+  width: 100%;
+}
+
+.litteral-expanded ion-label {
+  white-space: normal !important;
```

Do *not* use !important. Never.
I see, that it's not so easy to target that label since it might have more than one class and giving the ion-label itself a class is a hazzle aswell. But everything is better than !important.

Try this for instance:
```suggestion
 .litteral-expanded .sc-ion-label-ios-h,  .litteral-expanded .sc-ion-label-ios-s ios
  white-space: normal;
```
add more classes separated by comma if you find more...

---

Done

---

Keep in mind that ionic has two modes: ios and md, as described in this [documentation](https://ionicframework.com/docs/theming/platform-styles#ionic-modes).

[This line of code](https://github.com/geovistory/design-system/blob/4b4db3df740eb61695a636e772c0fbcb1d9350b4/packages/design-system-web/.storybook/preview-head.html#L239) defines, that storybook presents our components in ios mode:

```javascript
document.documentElement.setAttribute('mode', 'ios');
```

Because of this mode, ionic adds classes like `sc-ion-label-ios-h`. If the mode was `md`, ionic would add classes like `sc-ion-label-md-h`.

Thus you have to write:

```css
.litteral-expanded .sc-ion-label-ios-h,  .litteral-expanded .sc-ion-label-ios-s ios,
.litteral-expanded .sc-ion-label-md-h,  .litteral-expanded .sc-ion-label-md-s md {
  white-space: normal
}
```

... and test it with:

```javascript
document.documentElement.setAttribute('mode', 'md');
```

---

Oh I didn't know that! Thanks for your clarification!